### ISSUE TITLE
Labels not read out correctly by screen readers

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#9526]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=29659&p=150014#p150014)

"Certain cell labels are not being read out as "L{cellLabel}" by screen readers rather than the actual cell label.

![Screenshot 2024-07-05 at 14.21.09.png](https://forum.bryntum.com/download/file.php?id=29533)

The issue is in the default fields configuration on the Grid.column.Column class:

```
/**
 * The `aria-label` to use for this Column`s header element
 * @config {String} ariaLabel
 * @category Accessibility
 */
{
    name: "ariaLabel",
    defaultValue: "L{Column.columnLabel}"
},
/**
 * The `aria-label` to use for cells in this Column
 * @config {String} cellAriaLabel
 * @category Accessibility
 */
{
    name: "cellAriaLabel",
    defaultValue: "L{cellLabel}"
}

```

The defaultValue for ariaLabel uses full locale reference: L{Column.columnLabel} whereas the cellAriaLabel field is misconfigured. It should be L{Column.cellLabel}."

**Comment by marciogurka** [comment]
solved on https://github.com/bryntum/bryntum-suite/pull/10572
